pkg/oras/store.go

```diff
@@ -6,13 +6,15 @@ import (
 	orascontent "github.com/deislabs/oras/pkg/content"
 
 	"github.com/containerd/containerd/content"
+	"github.com/opencontainers/go-digest"
 	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
 )
 
 // ensure interface
 var (
 	_ content.Provider = &hybridStore{}
 	_ content.Ingester = &hybridStore{}
+	_ content.Store    = &hybridStore{}
```

Comment:
`content.Store` only is enough since it includes `Provider` and `Ingester`.

Reply:
done